src/index.js

```diff
@@ -57,11 +57,11 @@ function eventDrops(config = {}) {
                 );
 
             scales = getScales(dimensions, finalConfiguration, data);
-            const draw = drawer(svg, dimensions, scales, finalConfiguration);
-            draw(data);
+            this.draw = drawer(svg, dimensions, scales, finalConfiguration);
+            this.draw(data);
 
             if (finalConfiguration.zoomable) {
-                zoom(svg, dimensions, scales, finalConfiguration);
+                this.zoom = zoom(svg, dimensions, scales, finalConfiguration, data);
```

Comment:
Can you, if the chart is not zoomable, define `this.zoom` as an empty function `() => {}`? This way, it would always be a function, leading to an easier API.